nurse/local_generator.py

```diff
@@ -49,4 +49,11 @@ def time(self):
         if len(self._time) > 0:
             return -(np.asarray(self._time) - self._time[-1]) / 1000
         else:
-            return []
+            return np.array([], dtype=np.double)
+
+    @property
+    def timestamp(self):
```

Comment:
Do we want to divide this? Or leave it in pure timestamps?

Reply:
What I meant by the name "timestamp" is floating point seconds since epoch. If there's a better name, let's take it. I just wanted to distinguish from "time".

Reply:
Yes, but for some purposes the int is also possibly useful, such as checking to see which ones are new, but okay.

Reply:
Even as an integer, I would have to do difference tests, rather than check for strict equality. The landmarks in the time-series data—maximum and minimum volumes, for instance—are determined by a fit that can change infinitesimally as new data are added. I'm matching derived products, breath records, rather than the original time-series, so the matching is approximate.